src/common/src/DocumentViewer/DocumentRenderer.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Alert, Link, Paragraph } from '@amsterdam/asc-ui';
+import { DocumentRendererStyle } from './DocumentRendererStyles';
+
+type Props = {
+	uri: string;
+	contentType: string;
+};
+
+export const imageContentTypes: string[] = [
+	'jpg',
+	'jpeg',
+	'image/jpg',
+	'image/jpeg',
+	'png',
+	'image/png',
+	'bmp',
+	'image/bmp',
+	'tif',
+	'tiff',
+	'image/tif',
+	'image/tiff',
+];
+
+export const pdfContentTypes: string[] = ['application/pdf', 'pdf'];
```

Comment:
Have you considered making this a TS enum? In that way you could have type safety on the passed strings.

Reply:
Dome